packages/clay-date-picker/src/Hooks.ts

```diff
@@ -35,18 +35,34 @@ const useCurrentTime = (format: string) => {
 			.format(format)
 	);
 
-	function setCurrentTime(hours: number, minutes: number): void {
-		set(
-			moment()
+	function setCurrentTime(
+		hours: number | string,
+		minutes: number | string
+	): void {
+		const currentTime = [hours, minutes];
```

Comment:
Whats the advantage of using an array here? Wouldn't it be simpler to just do something like

```js
function setCurrentTime(
	hours: number | string,
	minutes: number | string
): void {
	if (typeof hours !== 'string') {
		hours = moment()
			.set('h', hours)
			.format('H');
	}

	if (typeof minutes !== 'string') {
		minutes = moment()
			.set('m', minutes)
			.format('m');
	}

	set(`${hours}:${minutes}`);
}

```

Reply:
Yeah this really is not necessary, I ended up putting it there to build other cases but I just forgot to remove that logic, thanks for that.